api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -42,11 +42,7 @@ public ResponseEntity<Void> groupAndPushSynchronizeAccessTasks() {
 
   @Override
   public ResponseEntity<Void> sendAccessExpirationEmails() {
-    var users = userService.getAllUsers();
-    log.info(
```

Comment:
moved logs to CloudTaskUserController, which is now where `maybeSendAccessTierExpirationEmails()` is called